packages/radix-vue/src/Accordion/AccordionHeader.vue

```diff
@@ -1,33 +1,34 @@
 <script setup lang="ts">
-import { inject } from "vue";
 import { PrimitiveH3, type PrimitiveProps } from "@/Primitive";
+import { inject } from "vue";
 
 interface AccordionHeaderProps extends PrimitiveProps {}
-const props = defineProps<AccordionHeaderProps>();
 
-import {
-  ACCORDION_IMPL_INJECTION_KEY,
-  type AccordionImplProvideValue,
-} from "./AccordionImpl.vue";
 import {
   ACCORDION_ITEM_INJECTION_KEY,
   type AccordionItemProvideValue,
 } from "./AccordionItem.vue";
+import {
+  ACCORDION_INJECTION_KEY,
+  type AccordionProvideValue,
+} from "./AccordionRoot.vue";
 
-const accordionImplInjectedValue = inject<AccordionImplProvideValue>(
-  ACCORDION_IMPL_INJECTION_KEY
-);
-const accordionItemInjectedValue = inject<AccordionItemProvideValue>(
+const injectedRoot = inject<AccordionProvideValue>(ACCORDION_INJECTION_KEY);
+const injectedItem = inject<AccordionItemProvideValue>(
   ACCORDION_ITEM_INJECTION_KEY
 );
+
+const props = withDefaults(defineProps<AccordionHeaderProps>(), {
+  asChild: false,
+});
```

Comment:
we extended AccordionHeaderProps with PrimitiveProps, so we dont need to declare asChild default to false